spec/requests/users_email_controller_spec.rb

```diff
@@ -9,11 +9,11 @@
   fab!(:moderator) { Fabricate(:moderator) }
 
   describe "#confirm-new-email" do
-    it 'redirects to login for signed out accounts' do
+    it 'does not redirect to login for signed out accounts, this route works fine as anon user' do
       get "/u/confirm-new-email/asdfasdf"
 
-      expect(response.status).to eq(302)
-      expect(response.redirect_url).to eq("http://test.localhost/login")
+      expect(response.status).to eq(200)
+      expect(response.redirect_url).not_to eq("http://test.localhost/login")
```

Comment:
Hmm if the `response.status` is `200`, we don't need this assertion anymore since a redirection will have a `response.status` of `301`

Reply:
Ok, will change